python/semantic_kernel/planners/function_calling_stepwise_planner/function_calling_stepwise_planner_result.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import sys
+
+if sys.version_info >= (3, 9):
+    from typing import Annotated
+else:
+    from typing_extensions import Annotated
+
+from semantic_kernel.contents.chat_history import ChatHistory
+from semantic_kernel.functions.kernel_function_decorator import kernel_function
+from semantic_kernel.kernel_pydantic import KernelBaseModel
+
+
+class FunctionCallingStepwisePlannerResult(KernelBaseModel):
+    """The result of the function calling stepwise planner"""
+
+    final_answer: str = ""
+    chat_history: ChatHistory = None
+    iterations: int = 0
+
+
+class UserInteraction:
+    """The Kernel Function used to interact with the user"""
+
+    @kernel_function(description="The final answer to return to the user", name="SendFinalAnswer")
```

Comment:
could we make this name smaller (less tokens?)

Reply:
You're reffering to the "SendFinalAnswer" kernel function name? "SendFinalAnswer" is 3 tokens, and we want to reduce it to say "SendAnswer" which is 2 tokens. Is it worth it?

Reply:
3 is fine, just looks long I guess...